spot-client/src/spot-remote/analytics/index.js

```diff
@@ -0,0 +1 @@
+import './middleware';
```

Comment:
I'm actually not a fan of an import causing side effects. I'd much prefer, stylistically, to have something import this and apply it explicitly. I can't justify completely how or why I feel this way, but it's a bit less surprising to me to have to manually initialize. What is your opinion?

Reply:
But just now I remembered how we’re now in a mode of moving everything into middleware: because I tried to be different from jitsi-meet. So nevermind, let’s do whatever jitsi meet does to make choices easier.

Reply:
yeah I agree, but I just don't know how to do any better without extra manual work. It seems that once the feature is imported it kind of enables the middleware.

Reply:
well, maybe I will come up with something eventually, as I'm trying to enable it only for Spot Remote and it was done by importing the module here:
https://github.com/jitsi/jitsi-meet-spot/blob/add-analytics-events2/spot-client/src/spot-remote/ui/views/remote-control.js#L15
Which is probably not good enough, because we will probably need the analytics for the `join-code-entry` as well

Reply:
I don't know it doesn't seem to be very urgent. Anyway I'm wondering if it would be okay to import the analytics again from `join-code-entry` until there is that bundle ? Seems like the easiest thing to do really

Reply:
Importing twice would not register the middleware twice.

Reply:
then I really kind of like it as it seems there's a bit of automagic in this approach as well